store/collections/fashion.js

```diff
@@ -1,6 +1,6 @@
 import { defaultFacetNames } from '../search';
 
-const fashionFacetNames = ['CREATOR'].concat(defaultFacetNames);
+const fashionFacetNames = ['CREATOR', 'proxy_dc_format', 'proxy_dcterms_medium', 'proxy_dc_type'].concat(defaultFacetNames);
```

Comment:
Shouldn't these new facets all be `.en`? They are on Portal.rb

Reply:
yes, this is causing some issues with the labelling I'll need to look into though.

Reply:
I've  pushed changes to escape the translation keys. This is only applied to the more facet dropdown elements. I've not applied the same convention of facet name -> translation key modifications on date facets or the standard dropdowns, as they won't contain these kind of keys for now.

An alternative approach to this may be additional nesting in the locale file, but this would move the translations to different depth depending on the actual facet name and seems inconsistent to me.